pkg/arduino_lis3dh/Makefile

```
@@ -0,0 +1,11 @@
+PKG_NAME=arduino_lis3dh
+PKG_URL=https://github.com/adafruit/Adafruit_LIS3DH.git
+PKG_VERSION=0f4cda36805570ca87f91d811227edff2958053b
+PKG_LICENSE=BSD
+
+.PHONY: all
+
+all:
+	"$(MAKE)" -C $(PKG_BUILDDIR) -f $(CURDIR)/Makefile.arduino_lis3dh
```

```suggestion
	"$(MAKE)" -C $(PKG_BUILDDIR) -f $(RIOTBASE)/Makefile.base MODULE=arduino_lis3dh
```

And you can remove the `Makefile.arduino_lis3dh`